src/client/components/DataManager/useFetch.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from "react";
```

Comment:
I enjoy how you split your code into smaller re-useable pieces, keep doing that!